Tribler/Core/APIImplementation/LaunchManyCore.py

```diff
@@ -117,10 +117,12 @@ def register(self, session, sesslock):
             if self.session.get_torrent_store():
                 from Tribler.Core.leveldbstore import LevelDbStore
                 self.torrent_store = LevelDbStore(self.session.get_torrent_store_dir())
+                assert self.torrent_store.get_db() is not None
```

Comment:
I think we shouldn't have assert statements in production code. We have unit tests for this (while I understand your reasoning here and why it might be helpful in this particular case). Now that we are handling all cases of the except when creating a leveldb store, I don't think we need these assert statements (if the database cannot be created, we always throw an exception now).

Reply:
Hmm, alright, will make the changes. Looking at the reporter log [#11858](http://reporter.tribler.org/stacktrace/11858), it seems Tribler was already running for 50 seconds so somehow it might have got through the constructor exception handling.